project-canvas/src/components/projects-view/TableSort.tsx

```diff
@@ -0,0 +1,204 @@
+import { useEffect, useState } from "react"
+import {
+  createStyles,
+  Table,
+  ScrollArea,
+  UnstyledButton,
+  Group,
+  Text,
+  Center,
+  TextInput,
+} from "@mantine/core"
+import { keys } from "@mantine/utils"
+import {
+  IconSelector,
+  IconChevronDown,
+  IconChevronUp,
+  IconSearch,
+  TablerIcon,
+} from "@tabler/icons"
+
+const useStyles = createStyles((theme) => ({
+  th: {
+    padding: "0 !important",
+  },
+
+  control: {
+    width: "100%",
+    padding: `${theme.spacing.xs}px ${theme.spacing.md}px`,
+
+    "&:hover": {
+      backgroundColor:
+        theme.colorScheme === "dark"
+          ? theme.colors.dark[6]
+          : theme.colors.gray[0],
+    },
+  },
+
+  icon: {
+    width: 21,
+    height: 21,
+    borderRadius: 21,
+  },
+}))
+
+export interface RowData {
```

Comment:
Typo in RawData and maybe we can find a better descriptive name. It's not clear what raw data is that, and it can get confusing.

Reply:
no typo, but found better names